src/widgets/dialogs/EmotePopup.cpp

```diff
@@ -168,13 +168,22 @@ EmotePopup::EmotePopup(QWidget *parent)
     searchRegex.setPatternOptions(QRegularExpression::CaseInsensitiveOption);
     QValidator *searchValidator = new QRegularExpressionValidator(searchRegex);
 
+    layout->setMargin(0);
+    layout->setSpacing(0);
+
```

Comment:
Why do we need these lines? I'm pretty sure this part is unnecessary, tested without these lines and the functionality was exactly the same.

Reply:
I believe the default margin in Qt is not 0, from my testing I noticed more space above/under the search widget. Also I basically took existing code from SearchPopup.cpp since I'm not really proficient with Qt (never worked with it before). https://github.com/Chatterino/chatterino2/blob/f4c82dfa59d5a5e9adda10d8317ac0eb229bc71c/src/widgets/helper/SearchPopup.cpp#L136-L184

Reply:
@zneix Are you fine with the reply for this? I do remember you saying you believed this was copy-pasted, which is confirmed to be true, but if there's a more noticeable space I can just go ahead and merge this with the below approval.